src/lib/airtable/request.js

```diff
@@ -508,18 +509,22 @@ export const updateCustomer = async (id, recordUpdates) => {
 
 // NONGENERATED: Edit customer
 export const editCustomer = async (customer) => {
-  try {
-    const resp = await fetch(`${process.env.REACT_APP_AIRTABLE_ENDPOINT_URL}/customers/edit`, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(customer)
-    })
-
-    console.log(resp);
-  } catch (err) {
-    console.log(err);
+  if (!customer.id) {
+    addToOfflineCustomer(customer, 'edits', customer.customerUpdate);
```

Comment:
Note to other reviewers: This workflow is consistent with the offline workflow we have for payments and meter reading functionality. This'll probably need to be added to / adjusted in the future once we have more functionality set up, but I'm okay with approving this for now and dealing with it down the road.